extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketClient.java

```diff
@@ -191,6 +196,11 @@ private void initializeFutures() {
      * @return      A valid url for websocket connections
      */
     protected String validifyUrl(String url) {
+        if (url == null) {
+            log.warn("No websocket url given. Using default of 'wss://dev.vantiq.com/api/v1/wsock/websocket'");
```

Comment:
I think we should not default this.  Given that we have many international customers & will have a wider presence in different clouds as we progress, I would prefer that this be a required parameter -- that is, the caller must provide a value.

Reply:
Alright. Should I just let it throw an NPE or InvalidArgument if they give a bad input?

Reply:
Invalid argument with explanation is fine.